### DISCUSSION TITLE
How to best display overall wallet balance (with synthetic USD)

### DISCUSSION THREAD

**Discussion by klochowicz** [get10101/10101#1208] [General]
I have recently added synthetic USD into our wallet balance, aware that it might have some shortcomings (as discussed with @holzeis recently on #1158). Some ideas also started circling on Twitter about it.

What I'd like to see (feel free to challenge this! ☺️) is the following
- make the *synthetic USD* stand out somehow (part of the USD balance is volatile, part is not - they're *not* the same). It also highlights our USP clearly, on the main screen.
There are many ways of doing is, and just adding it at the end was the quickest/dirtiest way of doing it 😅 Any ideas?

There are also quite a few design questions that are interesting here (especially when we unlocked synthUSD feature):

1. is it useful to have *all* our funds jumbled together in one currency (e.g. $5 on-chain, $5 off-chain, $5 in synthetic USD gives $15 in total)? What's problematic would be if the user wants to pay with $10 worth of funds, and realises that they can't do that easily, because of having essentially 3 "wallets" in the app.
2. What if the user wants to have their balance displayed in EUR or any other currency? do we display 8EUR + $5 or we display the synthetic USD in *approximate* value in EUR?
3. Should we display the amount locked in an open position in the wallet balance? (we kinda do this with synthetic-USD, because it's *not risky* due to being a short position with leverage of 1). Perhaps adding it to your sats balance would make sense too?)
4. How does it scale when we allow different kinds of positions? e.g. EUR/BTC. Synth EUR + Synth USD could coexist easily.

All in all, this would have made a great A/B experiment, so I'd like to increase the annoyance counter on not having one. #1166  note: it absolutely doesn't need to be Firebase, there were some other options in the past, or we could hand-craft one. What I'm just trying to say, it would be useful to have one to try UI features quickly and gathering feedback :)

**Comment by jooray** [comment]
1. Yes, I would put it together. At some point I would love to have the balanced merged into one lightning balance (maybe the way phoenix does it). There should be no on-chain balance that is separate from lightning. Splice-in and splice-out. Hopefully, some day :)
2. I would show balances in the currencies they are in (10000 sats, $5 synthetic usd) and then the whole value in EUR. 

4. It's all OK - display 10000 sats, $5 USD, €5 EUR, total value $15 / €15. Rooting for synthetic EUR soon!
